crt_portal/cts_forms/templates/forms/question_cards/single_question.html

```diff
@@ -0,0 +1,19 @@
+<div class="form-group">
+  {% if field.label %}
+    <h3><label
+      for="{{ field.id_for_label }}"
+      class="{{label_class}} margin-bottom-0"
+    >
+      {{ field.label }} {% if field.field.required != True %}<em>(Optional)</em>{% endif %}
```

Comment:
Small thought, we could handle this conditional with `if not field.field.required`